node/pkg/governor/mainnet_tokens_test.go

```diff
@@ -0,0 +1,108 @@
+package governor
+
+import (
+	"github.com/certusone/wormhole/node/pkg/vaa"
+	"github.com/stretchr/testify/assert"
+	"testing"
+)
+
+func TestTokenListSize(t *testing.T) {
+	tokenConfigEntries := tokenList()
+
+	/* Assuming that governed tokens will not go down over time,
+	   lets set a floor to avoid parsing or loading regressions */
+	assert.Greater(t, len(tokenConfigEntries), 122)
+}
+
+func TestTokenListFloor(t *testing.T) {
+	tokenConfigEntries := tokenList()
+
+	/* Assume that we will never have a floor price of zero,
+	   otherwise this would disable the value of the notional
+	   value limit for the token */
+	for _, tokenConfigEntry := range tokenConfigEntries {
+		testLabel := vaa.ChainID(tokenConfigEntry.chain).String() + ":" + tokenConfigEntry.symbol
+		t.Run(testLabel, func(t *testing.T) {
+			assert.Greater(t, tokenConfigEntry.price, float64(0))
+		})
+	}
+
+}
+
+func TestTokenListAddressSize(t *testing.T) {
+	tokenConfigEntries := tokenList()
+
+	/* Assume that token addresses must always be 32 bytes (64 chars) */
+	for _, tokenConfigEntry := range tokenConfigEntries {
+		testLabel := vaa.ChainID(tokenConfigEntry.chain).String() + ":" + tokenConfigEntry.symbol
+		t.Run(testLabel, func(t *testing.T) {
+			assert.Equal(t, len(tokenConfigEntry.addr), 64)
+		})
+	}
+}
+
+func TestTokenListChainTokensPresent(t *testing.T) {
+	tokenConfigEntries := tokenList()
+
+	chains := []uint16{1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 18}
+
+	/* Assume that all chains will have governed tokens */
+	for _, chain := range chains {
+		t.Run(vaa.ChainID(chain).String(), func(t *testing.T) {
+			found := false
+			for _, tokenConfigEntry := range tokenConfigEntries {
+				if tokenConfigEntry.chain == chain {
+					found = true
+					break
+				}
+			}
+
+			assert.Equal(t, found, true)
+		})
+	}
+}
+
+func TestTokenListTokenAddressDuplicates(t *testing.T) {
+	tokenConfigEntries := tokenList()
+
+	/* Assume that all governed token entry addresses won't include duplicates */
+	for x, tokenConfigEntry1 := range tokenConfigEntries {
+		for y, tokenConfigEntry2 := range tokenConfigEntries {
+			if x == y {
+				// don't flag duplicates at the same index
+				continue
+			}
+
+			assert.NotEqual(t, tokenConfigEntry1.addr, tokenConfigEntry2.addr)
+		}
+	}
```

Comment:
style nit: the idiomatic way to check for duplicates in go is to use a set:

```suggestion
	addrs := make(map[string]bool)
	for _, e := range tokenConfigEntries {
		assert.False(t, addrs[e.addr])
		addrs[e.addr] = true
	}
```

Here the assertion relies on the fact that [looking up a non-existent key in the map will return the zero value](https://go.dev/doc/effective_go#maps) (false for bools)